meson.build

```diff
@@ -0,0 +1,31 @@
+project('nwg-launchers', 'cpp',
+	default_options: ['cpp_std=c++17'],
+	license: 'GPL-3.0',
```

Comment:
@nwg-piotr regarding license, is it "GPL-3.0-only" or "GPL-3.0-or-later"? Those are the valid SPDX headers for GPL.

Reply:
I have no idea I just copied the name from the LICENSE file

Reply:
I didn't care much of the license, but I had GPL-3.0-or-later in mind.

Reply:
Cool, thanks! It helps with packaging to have it specified c: